src/main/kotlin/com/criptext/mail/scenes/mailbox/data/MailboxDataSource.kt

```diff
@@ -72,9 +72,9 @@ class MailboxDataSource(
             is MailboxRequest.SendMail -> SendMailWorker(
                     storage = storage,
                     accountDao = accountDao,
-                    signalClient = if(params.senderAccount == null) signalClient
-                                    else SignalClient.Default(SignalStoreCriptext(db, params.senderAccount)),
-                    activeAccount = params.senderAccount ?: activeAccount,
+                    signalClient = SignalClient.Default(SignalStoreCriptext(db, params.senderAccount)),
```

Comment:
Check if in `MailboxRequest.ResendEmails` you need signalClient var. Otherwise you can delete it.

Reply:
it's used